src/BenchmarkDotNet/Reports/BenchmarkReport.cs

```diff
@@ -47,6 +47,16 @@ public sealed class BenchmarkReport
                 ?? (IReadOnlyDictionary<string, Metric>)ImmutableDictionary<string, Metric>.Empty;
         }
 
+        public BenchmarkReport(BenchmarkCase benchmarkCase, GenerateResult generateResult, BuildResult buildResult)
```

Comment:
nit: it does not need to be public

```suggestion
        internal BenchmarkReport(BenchmarkCase benchmarkCase, GenerateResult generateResult, BuildResult buildResult)
```